src/Model.elm

```diff
@@ -219,18 +210,31 @@ subscriptions model =
 
 fetchRecord : RecordId -> Cmd Msg
```

Comment:
Why are these living in Model.elm and not in Kinto.elm? I may be missing something obvious, or you're possibly in the middle of migrating them there, but to me these should be fully part of the core API of a Kinto client library :)